test/FunctionalTests/Infrastructure/GrpcTestFixture.cs

```diff
@@ -45,6 +46,8 @@ public GrpcTestFixture()
                 .UseStartup<TStartup>();
 
             _server = new TestServer(builder);
+            // Temporary workaround, this should be added to the TestServer by default
+            _server.Features.Set<IHttpResponseStartFeature>(new TestHttpResponseStartFeature());
```

Comment:
I didn't know you could do this. I've been setting workaround features in the functional website's Startup.cs via middleware.

Reply:
I was wondering if this worked but seems like it didn't.